src/sink/wavefront.rs

```diff
@@ -34,6 +34,23 @@ lazy_static! {
     pub static ref WAVEFRONT_VALVE_OPEN: Arc<AtomicUsize> = Arc::new(AtomicUsize::new(0));
 }
 
+#[derive(Copy, Clone, Debug, Default, Serialize, Deserialize)]
+/// Controls which aggregrations will be padded
+pub struct PadControl {
+    /// DO pad SET Telemetry if the value is true, DO NOT pad if the value is
```

Comment:
As padding is seemingly unique to cernan can you put in an explanation about what we mean by padding?

Perhaps in the documation for the wavefront sink?

Reply:
Good idea. I'll update the wiki with the configuration option, padding discussion once this PR is merged. Work?

Reply:
Good for me.